packages/pluggableWidgets/toggle-buttons-native/webpack.config.dev.js

```diff
@@ -1 +1,59 @@
-module.exports = require("../../../configs/webpack.dev.native.js");
+const merge = require("webpack-merge");
+const baseConfig = require("@mendix/pluggable-widgets-tools/configs/webpack.native.config.js");
+
+const customConfig = {
+    module: {
+        rules: [
+            {
+                test: /\.tsx?$/,
+                loader: "ts-loader",
+                options: {
+                    compilerOptions: {
+                        target: "esnext"
+                    }
+                }
+            },
+            {
+                test: /\.jsx?$/,
+                exclude: /node_modules/,
+                use: {
+                    loader: "babel-loader",
+                    options: {
+                        cacheDirectory: true,
+                        presets: ["@babel/preset-react"],
+                        plugins: [
+                            ["@babel/plugin-proposal-class-properties", { loose: true }],
+                            ["@babel/plugin-transform-react-jsx", { pragma: "createElement" }]
+                        ]
+                    }
+                }
+            },
+            {
+                test: /\.jsx?$/,
```

Comment:
I think it is the only rule that is different from the base plugin. Why not just tweak a base config a bit, iso fully overriding all rules?

Reply:
By the way, we might want to include string-flow-types into the base webpack config, so that this override is not necessary at all. I see that metro-react-native [does it anyway](https://github.com/facebook/metro/blob/master/packages/metro-react-native-babel-preset/src/configs/main.js#L184)

Reply:
Simplified the configs